Capstone/API/APImanager.h

```diff
@@ -0,0 +1,16 @@
+//
+//  APImanager.h
```

Comment:
Super super small nit but lets try and keep class names as pascal case so the first letter of each word is capitalized so this should be APIManager

Reply:
Noted!